reactiveweb/src/ember-concurrency.ts

```diff
@@ -69,7 +65,7 @@ export function task<
 
   registerDestructor(state, () => state[TASK].cancelAll());
 
-  return destroyable as unknown as TaskInstance<Return>;
+  return destroyable as unknown as TaskInstance<Return> & { retry: () => void };
```

Comment:
I know in discord you said you might want to do the type differently -- but I think this is fine <3